src/components/Charts/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,35 @@
+import { LineChart as MantineLineChart } from '@mantine/charts';
+import { useViewportSize } from '@mantine/hooks';
+import { CHART_COLORS } from '@/theme';
+import { CustomLineData } from '../chart.types';
+import { chartStyles } from './styles';
+
+export default function LineChart({ data }: { data: CustomLineData }) {
+  const { width } = useViewportSize();
+  const isMobileView = width < 1024;

```

Comment:
No magic numbers, save it inside constants, btw dont we have breakpoints in the theme object?